spec/models/comment_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  # User.new(name: 'Jeff', email: 'jeff@hotmail.com', password: 'password1')
+  # Post.new(message: 'This is a post', user_id: 1)
+  #
+
+  subject do
+    user = FactoryGirl.create(:user)
+    post = FactoryGirl.create(:post, user: user)
+    described_class.new(comment: "This is a test", user_id: 1, post_id: 1)
```

Comment:
```suggestion
    described_class.new(comment: "This is a test", user: user, post: post)
```
can you try this and see if this works - i think the more rails way of doing things

Reply:
done!

Reply:
it works!